chainer/utils/experimental.py

```diff
@@ -0,0 +1,107 @@
+import chainer
+import inspect
+import warnings
+
+
+def experimental(api_name=None):
+    """A function for marking APIs as experimental.
+
+    The developer of an API can mark it as *experimental* by calling
+    this function. When users call experimental APIs, :class:`FutureWarning`
+    is issued.
+    The presentation of :class:`FutureWarning` is disabled by setting
+    ``chainer.disable_experimental_warning`` to ``True``,
+    which is ``False`` by default.
+
+    The basic usage is to call it in the function or method we want to
+    mark as experimental.
+
+    By default, if the caller is a function, its name is used as an API name.
+
+    >>> from chainer.utils import experiment
```

Comment:
Should this be `from chainer.utils import experimental`? Consider replacing usages of `experiment` with `experimental` below in docstring.